frontend/src/components/Sidebar.vue

```diff
@@ -128,6 +130,10 @@ export default {
     help() {
       this.$store.commit("showHover", "help");
     },
+
+    refresh:function(){
+      this.$router.go(0);  
```

Comment:
Using `go(0)` is inefficient, causing the whole page to be reloaded. You should use the `showHover` state to control the sidebar visibility.

Reply:
Good idea.

Reply:
Yes, and the current filebrowser also has this problem. When you route back from detail page(previewer) to list page (files or item listing), it always reload the list page, and then we can't come back to the exact location on the list page when we leave. that's inefficient. The previewer should work as a mask page, but currently it doesn't.

Reply:
The whole page is not reloaded when navigating from Preview back to Listing, in this case we have a partial reload where only information required to display the files is gathered from the server.